resources/public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,151 @@
+openapi: 3.0.0
+info:
+  title: National Insurance Test Support
+  contact: {}
+  version: '1.0'
+servers:
+- url: https://api.service.hmrc.gov.uk/
+  variables: {}
+paths:
+  /national-insurance-test-support/sa/{utr}/annual-summary/{taxYear}:
+    post:
+      tags:
+      - national-insurance-test-support
+      summary: Create National Insurance summary test data
+      description: Lets you create National Insurance contributions test data
+      operationId: CreateNationalInsurancesummarytestdata
+      parameters:
+      - name: utr
+        in: path
+        description: The 10 digit self-assessment UTR for the individual.
+        required: true
+        style: simple
+        schema:
+          type: string
+          example: '2234567890'
+      - name: taxYear
+        in: path
+        description: The tax year for the National Insurance contribution being created.
+        required: true
+        style: simple
+        schema:
+          pattern: ^[0-9]{4}-[0-9]{2}$
+          type: string
+          example: 2016-17
+      requestBody:
+        description: ''
+        content:
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/schemas/NationalInsuranceTestSupportSaAnnualSummaryrequest'
+              - example:
+                  scenario: HAPPY_PATH_1
+            example:
+              scenario: HAPPY_PATH_1
+        required: true
+      responses:
+        '201':
+          description: ''
+          headers: {}
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/NationalInsuranceTestSupportSaAnnualSummaryresponse'

```

Comment:
Please change to camel case i.e. `nationalInsuranceTestSupportSaAnnualSummaryResponse`. Thanks